templates/ta/_input.erb

```diff
@@ -0,0 +1,8 @@
+[<%= @name %>]
+<%
+@attributes.each do |key,val|
+-%>
```

Comment:
It's a silly change, but could you lump the `<%` and ruby code and `-%>` all on one line?